webpack/scenes/ContentViews/Create/CreateContentViewModal.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+import CreateContentViewForm from './CreateContentViewForm';
+
+const CreateContentViewModal = ({ show, setIsOpen }) => (
+  <Modal
+    title="Create Content View"
```

Comment:
Should check with Maria to make sure we're getting the capitalization right for the headings.

Reply:
+1 I do see it lower case on the button and the modal title in the mockup, we should change it to that or confirm with Maria that it can be changed.

Reply:
Updated to use proper case from mockups..